dev/com.ibm.ws.microprofile.config.1.2.services/bnd.bnd

```diff
@@ -0,0 +1,44 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.config.1.2.services
+Bundle-SymbolicName: com.ibm.ws.microprofile.config.1.2.services
+
+Bundle-Description:MicroProfile Configuration 1.2 Services, version ${bVersion}
+
+-dsannotations-inherit: true
+-dsannotations: com.ibm.ws.microprofile.config12.services.Config12ProviderResolverComponent
+
+Export-Package: \
+    com.ibm.ws.microprofile.config12.services
+
+Import-Package: \
+        org.eclipse.microprofile.config;version="[1.0,2)", \
+        org.eclipse.microprofile.config.spi;version="[1.2,2)", \
+        *
```

Comment:
Not sure what's going on with these imports.

1. This is the 1.2.services bundle, so it's only going to be used with 1.2
1. BND should cap the version at the next major version for you, so I don't think you need to declare these?